static/js/details.js

```diff
@@ -147,8 +147,12 @@ $(function () {
         document.querySelectorAll("#log tr").forEach(function(row) {
             var dt = moment.unix(row.dataset.dt).utc();
             format == "local" ? dt.local() : dt.tz(format);
+            var dateFormat = "MMM D"
+            if (moment().year() != dt.format("YYYY")) {
```

Comment:
* Move the `moment().year()` part outside the function, so it is not evaluated again for every row
* Instead of `dt.format("YYYY")`, do `dt.year()`

Reply:
Noted